CHANGELOG.md

```diff
@@ -496,7 +503,9 @@ Additions to existing modules
 
 * Added new proofs to `Data.List.Relation.Binary.Permutation.Propositional.Properties`:
   ```agda
-  product-↭ : product Preserves _↭_ ⟶ _≡_
+  product-↭   : product Preserves _↭_ ⟶ _≡_
+  catMaybes-↭ : xs ↭ ys → catMaybes xs ↭ catMaybes ys
+  mapMaybe-↭  : xs ↭ ys → mapMaybe f xs ↭ mapMaybe f ys
```

Comment:
Are these not (also) provable for the `Setoid` version?
If so, please add! (for me, that relation is conceptually prior to the `Propositional` one cf. #1761

Reply:
Indeed they are, but require another module/file that is more general than `Data.List.Relation.Binary.Permutation.Setoid.Properties`, i.e. without a fixed module parameter for setoid equality on `A`. 

I added this as `Data.List.Relation.Binary.Permutation.Setoid.Properties.Maybe`.

Reply:
Ugh! yes... a wrinkle that recurs through the library. 
But can't such proofs follow eg the model of `Data.List.Relation.Binary.Permutation.Setoid.Properties.map`?

Reply:
Unfortunately not: after flatting the maybes, you need to instantiate `catMaybes-↭` with type `B` instead of the (global) type `A`.

Reply:
Grrr. OK.

Reply:
See also #2397 this may require downstream refactoring...